content/en/opentelemetry/collector_exporter/_index.md

```diff
@@ -0,0 +1,222 @@
+       otel/opentelemetry-collector-contrib
+   ```
+
+
+[1]: https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags
+{{% /tab %}}
+{{% tab "Docker (other containers)" %}}
+
+To run the OpenTelemetry Collector as a Docker image and receive traces from other containers:
+
+1. Create a Docker network:
+
+    ```
+    docker network create <NETWORK_NAME>
+    ```
+
+2. Run the OpenTelemetry Collector and application containers as part of the same network.
+
+   ```
+   # Run the OpenTelemetry Collector
+   docker run -d --name opentelemetry-collector \
+       --network <NETWORK_NAME> \
+       --hostname $(hostname) \
+       -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
+       otel/opentelemetry-collector-contrib
+   ```
+
+   When running the application container, ensure that the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` is configured to use the appropriate hostname for the OpenTelemetry Collector. In the example below, this is `opentelemetry-collector`.
+
+   ```
+   # Run the application container
+   docker run -d --name app \
+       --network <NETWORK_NAME> \
+       --hostname $(hostname) \
+       -e OTEL_EXPORTER_OTLP_ENDPOINT=http://opentelemetry-collector:4317 \
+       company/app:latest
+   ```
+
+{{% /tab %}}
+{{% tab "Kubernetes (DaemonSet)" %}}
+
+Using a DaemonSet is the most common and recommended way to configure OpenTelemetry collection in a Kubernetes environment. 
+
+To deploy the OpenTelemetry Collector and Datadog Exporter in a Kubernetes infrastructure, use this [full example of configuring the OpenTelemetry Collector using the Datadog Exporter as a DaemonSet][1], including the application configuration example.
+
+Note especially some [essential configuration options from the example][2], which ensure the essential ports of the DaemonSet are exposed and accessible to your application:
+
+```yaml
+# ...
+        ports:
+        - containerPort: 4318 # default port for OpenTelemetry HTTP receiver.
+          hostPort: 4318
+        - containerPort: 4317 # default port for OpenTelemetry gRPC receiver.
+          hostPort: 4317
+        - containerPort: 8888  # Default endpoint for querying Collector observability metrics.
+# ...
+```
+
+If you do not need both the standard HTTP and gRPC ports for your application, it is fine to remove them.
+
+
+
+[1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2c32722/exporter/datadogexporter/examples/k8s-chart
+[2]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2c32722/exporter/datadogexporter/examples/k8s-chart/daemonset.yaml#L41-L46
+
+{{% /tab %}}
+{{% tab "Kubernetes (Gateway)" %}}
+
+To deploy the OpenTelemetry Collector and Datadog Exporter in a Kubernetes Gateway deployment, use this [full example of configuring the OpenTelemetry Collector using the Datadog Exporter as a DaemonSet][1], including the application configuration example.
```

Comment:
is this meant to link to the same examples as for Kubernetes DaemonSet? if so, then DaemonSet and Gateway have exactly the same instructions. is that intentional?